roles/pulp/templates/pulp.container.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Pulp Container
+After=network.target
+
+[Service]
```

Comment:
This needs to be a `[Container]` section for a quadlet, and all of the `Exec` commands you have below are unneeded. You can translate the options in the run command to top-level options. For example:

Instead of `-v /var/lib/pulp/settings:/etc/pulp:Z` you can do:

```
Volume=/var/lib/pulp/settings:/etc/pulp:Z
```

Take a look at https://github.com/ehelms/foreman-quadlet/blob/master/roles/candlepin/templates/candlepin.container.j2#L5-L26

Reply:
Oh yea, Isee you have it here -- https://github.com/ehelms/foreman-quadlet/pull/1/files#diff-b96d3b06048d3fbc532b73220a5bd877d7549e44a11ab70ae001f82a8b38b9b5

Move that file here instead inside the role and deploy it.